lib/metadataTypes/MetadataType.js

```diff
@@ -572,7 +590,12 @@ class MetadataType {
         let results = {};
         do {
             options.uri = this.paginate(options.uri, lastPage);
-            const response = await this.client.RestClient.get(options);
+            let response;
+            try {
+                response = await this.client.RestClient.get(options);
```

Comment:
if this actually fails, shouldnt it break something more than just logging?

Reply:
well, yes, just not within fuel which isn't great at producing errors one can handle... like with soap, when the API does not return a result, it just ends on a fatal error